api/src/services/uploads/uploads.ts

```diff
@@ -258,3 +258,42 @@ export const getUploadsByExpenditureCategory = async () => {
 
   return uploadsByExpenditureCategory
 }
+
+export const sendTreasuryReport: MutationResolvers['sendTreasuryReport'] =
```

Comment:
@Vikariusu can we add unit tests for this function? I wasn't sure if the comment below that you added is blocking your ability to do so.

Reply:
Just added testing